gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -143,9 +151,33 @@ properties:
       - Indefinite
       - Low Grade
       - No Dysplasia
+      - Mild
+      - Moderate
+      - Severe
       - Unknown
```

Comment:
Could you alphabetize these?  I think you just have to move No Dysplasia

Reply:
yes will do